webpack/ForemanTasks/Components/TasksTable/Components/ConfirmModal/ConfirmModal.js

```diff
@@ -16,32 +18,46 @@ export const ConfirmModal = ({
   uriQuery: query,
   setModalClosed,
   ...props
-}) => (
-  <ForemanModal
-    title={sprintf(__('%s Selected Tasks'), startCase(actionText))}
-    id={id}
-  >
-    {sprintf(
-      __(
-        `This will %(action)s %(number)s task(s), putting them in the %(state)s state. Are you sure?`
-      ),
-      { action: actionText, number: selectedRowsLen, state: actionState }
-    )}
-    <ForemanModal.Footer>
-      <Button onClick={setModalClosed}>{__('No')}</Button>
-      <Button
-        className="confirm-button"
-        bsStyle="primary"
+}) => {
+  if (actionType === FORCE_UNLOCK_MODAL) {
```

Comment:
It doesn't feel right, why not using the `ForceUnlockModal` from the parent instead?

Reply:
Can you explain what do you mean to use it from the parent?

Reply:
I guess there is a parent component that rendering `<ConfirmModal ... />` to render `ForemanModal` and `<ConfirmModal ... actionType={FORCE_UNLOCK_MODAL} />` to render `ForceUnlockModal`.

Why not just rendering `ForceUnlockModal` instead `<ConfirmModal ... actionType={FORCE_UNLOCK_MODAL} />`?

Reply:
The parent component doesn't have the info about actionType